nix/quota.nix

```diff
@@ -0,0 +1,20 @@
+{ mkDerivation, base, exceptions, fetchgit, mtl, QuickCheck, stdenv
+, test-framework, test-framework-quickcheck2, transformers
+}:
```

Comment:
Might be better to just add nix packaging for `quota` in its own repo, since I'm maintaining that anyway.

Reply:
By doing it this way, you pin to a specific commit of `quota`. Updating is easy:
`cabal2nix https://github.com/zenhack/haskell-quota > nix/quota.nix`.

Reply:
I think you can just do this:

https://github.com/haskell-miso/miso/blob/master/default.nix#L33-L37

...and it will fetch that revision and use the packaging in the repo. Unless I'm misunderstanding what that does? Still a bit new to nix.

Reply:
Yeah, I usually avoid import-from-derivation as it's slow and can cause problems on Hydra, but in this case it should be fine.